services/packagist/packagist-stars.service.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { optionalUrl } = require('../validators')
+const {
+  keywords,
+  BasePackagistService,
+  customServerDocumentationFragment,
+  cacheDocumentationFragment,
+} = require('./packagist-base')
+
+const schema = Joi.object({
+  package: Joi.object({
+    favers: Joi.number().required(),
```

Comment:
Can this be replaced by `nonNegativeInteger` in _validators.js_?

Reply:
Maybe ;) I'll try to find an example usage.

Reply:
Found some other places where this validator is used, tried it → works.